goas/cl/cl/src/balance.rs

```diff
@@ -17,6 +17,15 @@ pub struct Balance(pub RistrettoPoint);
 pub struct BalanceWitness(pub Scalar);
 
 impl Balance {
+    /// A commitment to zero, blinded by the provided balance witness
+    pub fn zero(blinding: BalanceWitness) -> Self {
+        Self(balance(
+            0,
+            curve25519_dalek::constants::RISTRETTO_BASEPOINT_POINT,
```

Comment:
nit: I guess we're using the ristretto basepoint because the unit does not matter anyway in a zero commitment, maybe put this in a comment? it's not straightforward from the signature what this does

Reply:
I've left a comment, we could optimize this further from `zero = 0 * G + r * H` to `zero = r * H`, this would save on a scalar multiplication + point addition.

This is done in STARK for the bundle proof but that's currently not the bottleneck so I'll punt on this for now, I'll leave the optimization as a comment.

Reply:
I was talking about readability, you don't immediately get what that ristretto basepoint is for from the function signature

Reply:
Does the comment I added help?